### ISSUE TITLE
YADCF 0.8.9 and DataTable 1.10 - language with url source

### ISSUE THREAD

**Issue by Sephyria** [vedmack/yadcf#292]
```
var tableDataTbl = $("#MyDataTableTest").DataTable({
    language: {
        url: '/js/DataTables/fr_FR.json'
    },
    scrollY: 400,
    paging : false,
    scrollCollapse: true,
    stateSave: true
});
```

This doesn't load properly filters on header and crash the table.
Placing the json content of that fr_FR.json directly into the table init solve it.

(Thanks to Daniel for this solution ^^)

**Comment by jmoransalama** [comment]
With YADCF 0.9.1 and DataTables 1.10.12:

**Comment by vedmack** [comment]
fixed in [0.9.2.beta.2](https://github.com/vedmack/yadcf/blob/d5dba790602738a234d4629596b1005c1d9fab56/src/jquery.dataTables.yadcf.js)

[see test page](https://jsfiddle.net/6c660hat/74/)
